pkg/serving/config_changes.go

```diff
@@ -152,6 +152,17 @@ func UpdateResources(template *servingv1alpha1.RevisionTemplateSpec, requestsRes
 	return nil
 }
 
+// UpdateServiceLabels updates the labels on a service
+func UpdateServiceLabels(service *servingv1alpha1.Service, vars map[string]string) error {
+	if service.ObjectMeta.Labels == nil {
+		service.ObjectMeta.Labels = make(map[string]string)
+	}
+	for key, value := range vars {
+		service.ObjectMeta.Labels[key] = value
+	}
```

Comment:
```
if value == "" {
  delete(service.ObjectMeta.Labels, key)
}

Reply:
Thanks, meant to ask about that.

Reply:
? is this comment still valid as we have now a dedicated mechanism to remove labels (in contrast to having empty labels, which is also allowed)

Reply:
No, this is no longer valid.